openapi/components/schemas/ProblemTypes/TooManyRequests.yaml

```
@@ -0,0 +1,34 @@
+type: object
+additionalProperties: true
+minProperties: 1
+description: RFC-7807 [problem details](https://tools.ietf.org/html/rfc7807) JSON object.
```

```suggestion
description: RFC-9457 [problem details](https://datatracker.ietf.org/doc/html/rfc9457) JSON object.
```
The RFC was superseded by 9457.

---

@adamaltman created an issue to fix that in all problems.